inference-engine/src/mkldnn_plugin/nodes/mkldnn_eltwise_node.cpp

```diff
@@ -1087,6 +1087,8 @@ void MKLDNNEltwiseNode::initSupportedPrimitiveDescriptors() {
         } else if (std::find(supportedPrecisions.begin(), supportedPrecisions.end(), prc) == supportedPrecisions.end()) {
             if (prc == Precision::U32 || prc == Precision::I64 || prc == Precision::U64) {
                 return Precision(Precision::I32);
```

Comment:
> @jiwaszki I just don't understand why do we need this change at all. CPU plug-in doesn't support FP64 precision in general and updating Eltwise node doesn't make any sense. Previous error message seems meaningful enough.
> My general point is that CPU nodes shouldn't implicitly handle precisions that a generally not supported by the plug-in.

@dmitry-gorokhov I see your point, so where should we make these changes? So what about these lines? I see converting of I64 to I32, isn't that a same thing? I think this approach of fixing precision issues was used for a while.

Reply:
Discussed offline.